app-backend/db/src/main/resources/migrations/V3__Add_keepInSourceBucket_to_uploads.sql

```diff
@@ -0,0 +1,4 @@
+-- Add a keepInSourceBucket boolean field to uploads table
+-- to indicate whether to use source tif bucket or not
+
+ALTER TABLE uploads ADD COLUMN keep_in_source_bucket BOOLEAN DEFAULT false NOT NULL;
```

Comment:
I think this needs to be `public.uploads`

Reply:
@notthatbreezy ah, good catch! This has been addressed in [47f59c8](https://github.com/raster-foundry/raster-foundry/pull/4997/commits/47f59c892247fba04b0fda7980d07b4266f56602)